projects/patches/draco-fix-install.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/cmake/draco_install.cmake b/cmake/draco_install.cmake
+index f7e1153..3be1ba1 100644
+--- a/cmake/draco_install.cmake
```

Comment:
Can you retrieve patches from f3d repo to avoid duplication?

Reply:
no, f3d is not even cloned when the patches are needed.